added scroll to top

gurrrung opened the pull request:
closes #14 

Added scroll to top button which matches the theme and gives a smooth UX.

![ezgif-4-f74adbef463d](https://user-images.githubusercontent.com/30543444/88212166-6e7a4180-cc74-11ea-94d4-1f40f86b9ee0.gif)

abhishek97 commented:
Add a Up icon to the left, that'll look better imo. Otherwise look great! Thanks!

gurrrung commented:
Feedback:
1) Added the icon as suggested (used standard fontawesome)
2) Now the icon only appears when user scrolls down. (this is what most websites do thus)
3) Added fade animation

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/30543444/88400604-372ba200-cde6-11ea-899e-e55466bfece3.gif)
